lang/test/org/partiql/lang/eval/EvaluatorStaticTypeTests.kt

```diff
@@ -137,28 +137,6 @@ class EvaluatorStaticTypeTests {
             "pathUnpivotEmptyStruct3",
             "projectOfUnpivotPath",
 
-            // Aggregates not supported by STIR
```

Comment:
While these tests are now succeeding, also need some tests in [StaticTypeInferenceVisitorTransformTest.kt](https://github.com/partiql/partiql-lang-kotlin/blob/main/lang/test/org/partiql/lang/eval/visitors/StaticTypeInferenceVisitorTransformTest.kt).